### PR TITLE
add extra tests for multimodal combnation

### PR DESCRIPTION
**Pull request by wanliAlex**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
add extra tests for multimodal combination

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

File: tests/tensor_search/test_multimodal_tensor_combination.py [hunk 1]
```diff
@@ -229,12 +238,144 @@ def test_multimodal_tensor_combination_tensor_value(self):
...
 
-        expected_tensor = np.mean([text_tensor_1 * 0.32, text_tensor_2 * 0, image_tensor_1 * -0.48, image_tensor_2 * 1.34], axis = 0)
+        expected_tensor = np.mean(
+            [text_tensor_1 * 0.32, text_tensor_2 * 0, image_tensor_1 * -0.48, image_tensor_2 * 1.34], axis=0)
         assert np.allclose(combo_tensor_1, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_2, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_3, expected_tensor, atol=1e-5)
         assert np.allclose(combo_tensor_4, expected_tensor, atol=1e-5)
 
+    def test_multimodal_tensor_combination_tensor_value_different_mappings(self):
```
Comment (pandu-k) [inline tests/tensor_search/test_multimodal_tensor_combination.py]: This is a good test. Can we do another one where each mapping is a completely different (no shared subfields)?

Comment (wanliAlex) [reply tests/tensor_search/test_multimodal_tensor_combination.py]: added

**Review by pandu-k — CHANGES_REQUESTED** [review]
Looks good. Can you please merge mainline back into this, (and use the the new AddDocsParam in add_docs)

**Comment by farshidz** [comment]
Closing as stale

**Pull request closed without merging**